documentation/API_documentation/QualityByDesign_User_Story.md

```diff
@@ -0,0 +1,15 @@
+| **_Begins when_**               | The customer application creates a QbD session using an application profile, device identifier or 3-legged token, and callback URI. The CSP provides a session ID in the response.                                                                                                                                                                                                                                                                                                                                                  |
+| **_Step 1_**                    | The customer application periodically sends its network KPIs for its session to the CSP.                                                                                                                                                                                                                                                                                                                                                                                                                                            |
+| **_Step 2_**                    | The CSP calculates a QbD score using KPIs and the application profile and sends it to the customer application via the callback URI provided.                                                                                                                                                                                                                                                                                                                                                                                       |
+| **_Step 3_**                    | The CSP sends asynchronous updates with the current score. The CSP can optionally include additional information to help diagnose and correct an issue when the score falls below acceptable thresholds.                                                                                                                                                                                                                                                                                                                            |
+| **_Step 4_**                    | When possible, the customer application takes corrective action.                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
```

Comment:
I think this should rather be "The customer application decides whether to take corrective action"

Reply:
sure, I can change it to that. The point I was trying to make is that in some scenarios the issue may not be in the application, but lies in the network operator's domain. In that case there could be a resolution that takes place on the network and the application does not need to do anything.